src/Components/Notifications/NotificationsList.tsx

```diff
@@ -102,19 +111,29 @@ export const NotificationsListFragmentContainer = createPaginationContainer(
   }
 )
 
-export const NotificationsListQueryRenderer = () => {
+interface NotificationsListQueryRendererProps {
+  type: NotificationType
```

Comment:
**question**: do you think it might be better to use an **automatically generated enum**?

```javascript
interface NotificationsListQueryRendererProps {
  type?: NotificationTypesEnum
}

...

<NotificationsListQueryRenderer type="ARTWORK_PUBLISHED" /> // All tab
<NotificationsListQueryRenderer type="ARTWORK_ALERT" /> // Alerts tab
<NotificationsListQueryRenderer type="ARTWORK_PUBLISHED" /> // Other supported types
```